src/main/resources/twilio-php/options/optionMethodArgs.mustache

```diff
@@ -1,3 +1,6 @@
-{{#vendorExtensions.optionFileParams}}{{^isArray}}
-        {{dataType}} ${{paramName}} = Values::NONE{{vendorExtensions.optionFileSeparator}}{{/isArray}}{{#isArray}}
-        array ${{paramName}} = Values::ARRAY_NONE{{vendorExtensions.optionFileSeparator}}{{/isArray}}{{/vendorExtensions.optionFileParams}}
+{{#vendorExtensions.optionFileParams}}{{#isString}}{{^isArray}}{{^isDate}}{{^isDateTime}}
```

Comment:
nit: if these conditions  i.e iString, isArray, isDate, isDateTime. are mutually exclusive. Then you can remove the `^` conditionality.

Reply:
addressed